x-pack/plugins/spaces/public/views/nav_control/nav_control.js

```diff
@@ -25,8 +25,13 @@ const module = uiModules.get('spaces_nav', ['kibana']);
 
 let spacesManager;
 
-module.controller('spacesNavController', ($scope, $http, chrome, activeSpace, spaceSelectorURL) => {
+module.controller('spacesNavController', ($scope, $http, chrome, spacesEnabled, activeSpace) => {
+  if (!spacesEnabled) {
```

Comment:
Do we need this? I thought that we didn't load any of the plugin's client-side code when it's disabled.

Reply:
When I was testing, it appeared to still load assets, but that was with dev auto-reloading running. I'll retest without that and verify